Da login fix

Syphontwo opened the pull request:
The changes in this branch enable users to put in a username and password as non-secure strings when connecting to DA servers.  I originally forked in order to get the functionality added by the Opc.Da.Url DaClient constructor which is not present in the NuGet package, but ended up making this small change and thought it would be good to post.

Syphontwo committed:
Added login using credentials

DaClient has a new constructor overload which includes the option of passing username and password.  These are used to build System.Net.NetworkCredential which in turn makes the Opc.ConnectData for the Opc.Da.server

Syphontwo committed:
Connection method refactor

Pulled out the main bits of the connect method into a private function.  This allows both the interface Connect and the obejct specific conenct to use the same logic for all except the one line which changes.

Syphontwo commented:
Not sure why appveyor says the build failed... I'm using this library as written right now.

Looking to the appveyor build, it looks like the failed items are nothing that I have changed.  This leads me to believe the current master fails as well.

jmbeach commented:
Hi Syphontwo, Thanks for the contribution! I understand why you would think that. However, the error is actually caused by a tool we use that enforces certain style guidelines.

Please review our [getting started guide](https://github.com/hylasoft-usa/cs-boilerplate) to see how to properly build locally to match the way we compile on appveyor.

Syphontwo commented:
Thanks for the info, I'm working on getting this all setup now.  Sorry I missed that part of the documentation!

Not sure exactly when I'll be able to get it rolled over, but hopefully it will be today or tomorrow.

Syphontwo committed:
Fixing to pass code style checks

setting up the cs-boilerplate correctly this time.

Syphontwo committed:
Fixing to pass code style checks

hit commit before seelcting this file, my bad

Syphontwo committed:
Merge branch 'DALoginFix' of https://github.com/Syphontwo/h-opc into DALoginFix

Merging back with master

Syphontwo committed:
removing non-relevant items

Syphontwo commented:
There we go!  Passing no problem.

The key issue was no space between the end of the `CheckResult(IResult result, string tag)` function and the `/// <summary>` of the `Connect(string userName, string password)` function.  Other than that, all the other work was simply getting grunt to run.  It was good to learn something new!

jmbeach committed:
remove extra gitignore and fix test project

Removed the gitignore inside the h-opc folder because there is already one in the directory above it.

Removed App.config from the test project because it is no longer part of the project and was causing the build to fail.

Syphontwo committed:
Merge pull request #1 from jmbeach/DALoginFix

remove extra gitignore and fix test project
